src/App.tsx

```
@@ -56,16 +57,61 @@ function Home() {
+            resize: "none"
+          }}
+        />
+      </div>
+      <div 
+        style={{ flex: 1, padding: "10px", borderLeft: "1px solid #ccc", boxSizing: "border-box" }}
+        role="region"
+        aria-label="Markdown preview"  
+      >
+        <h1>Markdown Preview</h1>
+        <div
+          dangerouslySetInnerHTML={{ __html: marked(text) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)